main.css

```diff
@@ -0,0 +1,248 @@
+     box-shadow: 0 4px 2px rgba(0,0,0,.05);
+     /* opacity: 0; */
+}
+
+
+.title {
+    display: inline-block;
+    width: 30%;
+    font-weight: bold;
+    font-size: 1.2rem;
+}
+
+.text {
+    width: 35%;
+    padding: .5rem;
+    font-size: 1rem;
+}
+
+.submit-btn {
+    width: 60px;
+    height: 30px;
+    padding: .5rem;
+    margin-top: -5rem;
+
+}
+
+.container {
+    display: flex;
+    flex-wrap: wrap;
+    width: 100%;
+    margin-top: 100px;
+}
+
+/* styling the video container element */
+.box1 {
+    width: 70%;
+    order: 0;
+    padding: 2rem 6rem;
+
+    /* border: 1px solid green; */
+}
+
+/* styling the video player and its description and buttons */
+
+.video-info {
+    margin: 1rem;
+}
+
+.video-title {
+    margin: .5rem 0;
+}
+
+.video-btns {
+    display: flex;
+    flex-wrap: wrap;
+    padding: .5rem 0;
+    justify-content: space-between;
+}
+
+.video-views {
+    font-weight: 400;
+    color: gray;
+    display: inline-block;
+}
+
+.video-interaction {
+    margin: .2rem;
+    color: gray;
+    cursor: pointer;
+}
+
+/* styling the description section */
+
+.description {
+    display: flex;
+    flex-wrap: wrap;
+    width: 100%;
+    padding: 1rem;
+    justify-content: space-between;
+}
+
+.avatar {
+    border-radius: 1000rem;
+    display: inline-block;
+    /* border: 1px solid red; */
+    order: 0;
+}
+
+.image {
+    max-width: 100%;
+    max-height: 100%;
+    height: 5rem;
+    width: 5rem;
+    border-radius: 1000rem;
+    padding: .5rem;
+}
+
+
+.details {
+    /* display: inline-block; */
+    /* border: 1px solid purple; */
+    order: 1;
+    width: 72%;
+    padding: .5rem 1rem;
+}
+
+.subscribe {
+    width: 170px;
+    height: 45px;
+    order: 2;
+    background-color: red;
+    color: white;
+    font-size: 1rem;
+    /* font-weight: 600; */
+    text-transform: uppercase;
+    border: none;
+    border-radius: 2px;
+    cursor: pointer;
+    transition: all .1s;
+}
+
+.subscribe:active {
+    background-color: gray;
+    color: white;
+}
+
+.topic {
+    order: 3;
+    margin: 2rem auto;
+}
+
+
+/* styling the aside bar */
+.box2 {
+    width: 30%;
+    order: 1;
+    padding: .5rem;
+    /* border-left: 1px solid gray; */
+    /* border: 1px solid blue; */
+}
+
+.thumbnail {
+    width: 100%;
+    /* border-bottom: 1px solid gray; */
+    padding: 1rem;
+    display: flex;
+}
+
+.thumbnail-img {
+    margin-right: 8px;
+    height: 94px;
+    width: 168px;
+    cursor: pointer;
+}
+
+.thumbnail-div {
+    display: inline-block;
+    height: 100%;
+    padding: .5rem;
+}
+
+.thumbnail-title {
+    display: inline-block;
+    cursor: pointer;
+    margin: .2rem 0;
+}
+
+.thumbnail-text,
+.thumbnail-views {
+    color: gray;
+    font-weight: 500;
+    cursor: pointer;
+    margin: .2rem 0;
+}
+
+/* styling the comments section and its components */
+.box3 {
+    width: 70%;
+    order: 2;
+    padding: 1rem 6rem;
+
+    /* border: 1px solid purple; */
+}
+
+.comment {
+    display: flex;
```

Comment:
I've noticed `display: flex` and `flex-wrap: wrap` display together a lot, maybe you can make a class out of it. Would also help someone identify flex containers without even having to look at CSS.

Message me if you do not understand.